packages/@aws-cdk/aws-ec2/lib/vpc.ts

```diff
@@ -162,22 +162,40 @@ export enum SubnetType {
  */
 export interface SubnetSelection {
   /**
-   * Place the instances in the subnets of the given type
+   * Select all subnets of the given type
    *
-   * At most one of `subnetType` and `subnetName` can be supplied.
+   * At most one of `subnetType` and `subnetGroupName` can be supplied.
```

Comment:
Should this be a union-like instead?

Reply:
Maybe. Bit hard to change right now, and then the right thing to make union-like is the whole `subnetSelection` object. I predict there will be many more subnet selection requirements in the future.